lib/path_funcs.sh

```diff
@@ -2,40 +2,40 @@ function platform_tools_path() {
   echo "${build_path}/.platform_tools"
 }
 
-function erlang_path() {
```

Comment:
i added prefixes to the names of these functions to make it easier to determine what's "build-time" (when `bin/compile` runs) and what's "runtime" (values present at runtime when the app actually runs).